project-base/kubernetes/deployments/rabbitmq.yml

```diff
@@ -0,0 +1,22 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+    name: rabbitmq
+    labels:
+        app: rabbitmq
+spec:
+    replicas: 1
+    template:
+        metadata:
+            name: rabbitmq
+            labels:
+                app: rabbitmq
+        spec:
+            containers:
```

Comment:
Other deployment manifests have ports defined for their containers. The definition is purely informational [according to the API reference](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.14/#container-v1-core)

>`ports` List of ports to expose from the container. Exposing a port here gives the system additional information about the network connections a container uses, but is primarily informational. Not specifying a port here DOES NOT prevent that port from being exposed. Any port which is listening on the default "0.0.0.0" address inside a container will be accessible from the network. Cannot be updated.

[Ismail Yenigül recommends to define the ports in the deplyoments](https://medium.com/faun/should-i-configure-the-ports-in-kubernetes-deployment-c6b3817e495) and most of deployment manifest examples define them as well:
>If you don’t configure ports in Kubernetes deployment, you can still access to the ports using Pod IP inside the cluster. [...] But it is good to configure. It will help you or others to understand the deployment configuration better.

Reply:
If I understand Ismail correctly, he stated the ports in Deployments are really informational (like Docker EXPOSE) and ports should be in `Services`.
Based on 
> As result, container ports is just informative and The `service` should be used to solve this problem!

Reply:
Then please, remove the ports from other deployments. It would be confusing when the information is sometimes included in the deployment and sometime it's not.

From what I've read it seems that it's generally preferable to have the info in deployments as well. I wouldn't mind not having it there though. I would mind having it inconsistent.

Reply:
That's reasonable. 
I added ports into the container in deployment in 6d1247cba3eb809123bad8e0f3fa136ac0a82e41 so it's easier to upgrade and, as you say, generally preferable.